docs/auth_index.md

```diff
@@ -25,21 +25,26 @@ Each person is a user in the system. To log in as a specific user, authenticate
 
 ### Owner
 
-The first user to log in to Home Assistant will be marked as the owner. This user is able to manage users.
+The user that is created during onboarding will be marked as "owner". The owner is able to manage other users and will always have access to all permissions.
```

Comment:
Owner permission piece is here?

Reply:
Can you show examples the permissions working? Because i am lost in where i need the configure this feature.

The Example in link a head didn't help me so much.

Reply:
@Gabrieleduardo94 

First at all, please do not comment in closed pull request. Secondly, developer document's audience is not user, but developer. At last, permission is a feature still under development, there is no end-user facing document yet.